installation/windows.md

```diff
@@ -74,25 +74,25 @@ Make sure to provide a valid Windows configuration with the installation, a samp
 
 ## Migration to Fluent Bit
 
-From version 1.9.0, `td-agent-bit` is a deprecated package and will be removed in the future.
+From version 2.0.0, `td-agent-bit` is a deprecated package and will be removed in the future.
 The correct package name to use now is `fluent-bit`.
 Both are currently provided to allow migration.
```

Comment:
Ah, missed this page when I updated the others. Need to clarify only the new one is now provided from 1.9.9 onwards.